modules/related-posts/jetpack-related-posts.php

```diff
@@ -866,7 +866,52 @@ protected function _action_frontend_init_ajax( array $excludes ) {
 					'classes'  => array()
 				),
 			);
-			$options['size'] = 3;
+
+			for ( $total = 0; $total < $options['size'] - 3; $total++ ) {
+				$related_posts[] = $related_posts[ $total ];
+			}
+
+			$current_post = get_post();
+
+			/** This filter is already documented in modules/related-posts/jetpack-related-posts.php */
+			$excluded_posts = apply_filters( 'jetpack_relatedposts_filter_exclude_post_ids', array( $current_post->ID ) );
```

Comment:
If someone is using this filter, this will be overwritten so that it might show the current post again.  Should we `array_merge` or something?